code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Concepts.hs

```diff
@@ -19,4 +19,4 @@ acronyms = [assumption, centreMass, dataDefn, gamePhysics, genDefn, goalStmt, in
 
 centreMass, gamePhysics :: CI
 centreMass  = commonIdeaWithDict "centreMass"  (ctrOfMass ^. term) "CM" [physics]
-gamePhysics = commonIdeaWithDict "gamePhysics" (pn "game physics library") "Game Physics" [physics]
+gamePhysics = commonIdeaWithDict "gamePhysics" (pn "Game Physics Library") "Game Physics" [physics]
```

Comment:
This is wrong - it should be all lower case here, and combinators to capitalize should be used at the use-site.